tests/test_agents.py

```diff
@@ -55,27 +62,60 @@ def setUp(self):
         self.agent_list: list[elf_agent.Agent] = []
         for agent_id, policy_name in enumerate(agent_policies):
             if policy_name == "random_agent":
-                example_agent = import_module(f"elfpy.agents.policies.{policy_name}").Policy(
+                example_agent = RandomAgent(
                     rng=np.random.default_rng(seed=1234),
                     trade_chance=1.0,
                     wallet_address=agent_id,
-                    budget=1_000,
+                    budget=FixedPoint("1_000.0"),
                 )
-            else:
-                example_agent = import_module(f"elfpy.agents.policies.{policy_name}").Policy(
-                    wallet_address=agent_id, budget=1_000
+            elif policy_name == "lp_and_withdraw":
```

Comment:
oh god, I hate the elif's. I think we can improve this with a collection of BotInfo objects maybe

Reply:
Yeah agreed see my reply to the other comment